package.json

```diff
@@ -21,6 +21,7 @@
     "@react-native-community/blur": "^3.6.0",
     "@react-native-community/slider": "^1.1.4",
     "@react-native-community/toolbar-android": "^0.1.0-rc.2",
+    "@rsksmart/rns-resolver.js": "^1.0.0",
```

Comment:
Whats the diff with this one?
https://github.com/rnsdomains/rns-resolver/

Reply:
The one you mentioned is the original implementation that is heavy. It contained a dependency to web3 and was 232kb. This new version is 35kb and does not have that dependency. @ilanolkies finished up a week or so ago and will be maintained in the future.

Reply:
:)

Reply:
@patogallaiovlabs that one is a smart contract repo, not a js lib. This is the repo for that dep https://github.com/rsksmart/rns-resolver.js/